tests/test_base_virtual_machine_factory.cpp

```diff
@@ -15,6 +15,7 @@
  *
  */
 
+#include <multipass/network_interface_info.h>
```

Comment:
We need to leave this include, because `NetworkInterfaceInfo` is forward-declared in `base_virtual_machine_factory.h`. I think it's better to leave it forward-declared in the header; including here does not harm because it's a test. What do you think?

Reply:
OK, sure, I did not see that.